### DISCUSSION TITLE
Setting up Jamulus with a Macbook Pro and a Yamah AG06 mixer interface.

### DISCUSSION THREAD

**Discussion by Stewbow** [jamulussoftware/jamulus#3179] [General]
Hi All
Only just started using Jamulus, sorry if its an obvious answer!!
I have a Mac Book Pro and have just bought a Yamaha AG06 Mixer interface.
I was listening to what I was playing through the headphone out on the mixer which apparently breaks the Golden Rule.
My question is where should I be plugging my headphones into?
If I plug into the Mac I can't hear anything.
Are there some settings that I need to change?
any advice would be greatly appreciated.

many thanks

**Comment by pljones** [comment]
You need your audio in from your mixer feeding Jamulus.  Jamulus output should return to the mixer.

You should _hear nothing_ whilst using Jamulus, **unless** you're connected to a server.

You'll need to set the mixer _not_ to feed the input mix to the headphones but feed them the return mix instead.

**Reply by Stewbow** [reply]
many thanks, I'll have ago

**Comment by chrisrimple** [comment]
You'll likely have to adjust the Mixer settings.  By default, it's probably routing all input to Headphones, so you're hearing it "live".  What you want instead is to route USB (aka, the Mac) to Headphones, so you only hear what's coming from the Jamulus server.  My [Remote Band Rehearsals doc](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) may be helpful.
